### DISCUSSION TITLE
Seed storage for local development?

### DISCUSSION THREAD

**Discussion by madeleineostoja** [supabase/supabase#9248] [Questions]
I'm trying to get local development with Supabase setup properly after trying a while ago, giving up and using a staging project, then blowing past free limits on that staging project.

The biggest roadblocks are with auth users and storage. I think I've figured out seeding auth users properly, but is there a way I can seed local storage buckets? As I understand it Supabase uses a local folder somewhere when running locally, can I seed data into that somehow? Or re-use a docker container with `supabase start` that has storage data in it?

**Comment by fenos** [comment]
Hi @madeleineostoja 

Sure you can seed your local storage-api environment.
I recommend you create a simple script in your project which simply makes a post request to the `upload` endpoint passing a local file in the body as described here:

https://supabase.com/docs/reference/storage/usage#upload-a-new-object

Feel free to reach out if you any specific issues
